2-mandatory/3-financial-times.js

```diff
@@ -23,59 +42,77 @@ function titleWithFewestWords(allArticleTitles) {
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
 function headlinesWithNumbers(allArticleTitles) {
-    // TODO
+  let arrayNumTittle = [];
+  let condition = ["1", "2", "3", "4", "5", "6", "7", "8", "9", "0"];
+  for (let title of allArticleTitles) {
+    if (condition.some((num) => title.includes(num))) {
+      arrayNumTittle.push(title);
+    }
+  }
+  return arrayNumTittle;
```

Comment:
This works well. You could also consider using regex to determine whether a number is present. https://www.w3schools.com/js/js_regexp.asp

Reply:
Thank you